tests/raft_scenarios/check_quorum_2

```diff
@@ -0,0 +1,28 @@
+start_node,0
+swap_nodes,2,in,1,2
+emit_signature,2
+
+dispatch_all
+periodic_all,10
+dispatch_all
+periodic_all,10
+dispatch_all
+
+swap_nodes,2,out,1,2
+
+disconnect_node,0
+
+periodic_one,0,100
+periodic_one,1,100
+dispatch_all
```

Comment:
Add comments and asserts if possible. We retire nodes 1 and 2, but 0 fails to replicate this update to the other nodes.